app/models/funding_widget.rb

```diff
@@ -0,0 +1,218 @@
+class FundingWidget
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  SUBJECTS = {
```

Comment:
Worth moving this to the translations file so its not polluting the model, then you can just have a set of keys here?